app/_src/gateway/plugin-development/quickstart.md

```diff
@@ -0,0 +1,679 @@
+---
+title: Quick Start
+book: plugin_dev
+chapter: 2
+---
+
+The following instructions are written to help you quickly build, test, and run your first 
+{{site.base_gateway}} custom plugin with [Lua](http://www.lua.org/). The remaining pages in 
+this section will provide details on other advanced topics related to 
+plugin development and best practices.
+
+### Prerequisites
+
+* [Docker](https://docs.docker.com/get-docker/) (or Docker equivalent) is used to run {{site.base_gateway}} and test code
+* [curl](https://curl.se/) is used to download web resources. `curl` is pre-installed on most systems.
+* [git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) is used to install and update software on the host machine.
+
+1. **Initialize a new plugin repository**
```

Comment:
Can we use H3 tags rather than a numbered list? It gives us a table of contents on the right hand side and gains us some more horizontal width

Reply:
I see what you're aiming for and makes sense.  Thoughts:
* We use numbered lists in other guided tutorials https://docs.konghq.com/gateway/latest/get-started/
* I think numbers can be helpful to reference steps, but for readers and editors 

Is there any way to make the numbered list render in the TOC?

Reply:
I don't think it's possible to render numbered lists in the TOC as there's no demarcation of what is a header and what's content. The get started numbers make sense as it's a short page.

I don't feel strongly about this. Even less so as the page is being split in to chapters

Reply:
I will do some experiments as I refactor the breakout and see what looks best.